app/src/main/java/vn/realtest/stock/justanewproject/Adapter/SearchAdapter.java

```diff
@@ -68,25 +75,61 @@ public void onBindViewHolder(SearchViewHolder holder, final int position) {
 
         switch (check_rate(searchData.getStock_rate())) {
             case UP:
-                holder.stock_rate.setBackgroundColor(increase_value); break;
+                holder.stock_rate.setTextColor(increase_value);
+                holder.stock_value.setTextColor(increase_value);
+                holder.stock_name.setTextColor(increase_value);
+                break;
             case DOWN:
-                holder.stock_rate.setBackgroundColor(decrease_value); break;
+                holder.stock_rate.setTextColor(decrease_value);
+                holder.stock_value.setTextColor(decrease_value);
+                holder.stock_name.setTextColor(decrease_value);
+                break;
             case NOTCHANGED:
+                holder.stock_rate.setTextColor(ref_value);
+                holder.stock_value.setTextColor(ref_value);
+                holder.stock_name.setTextColor(ref_value);
                 break;
         }
         holder.cv_search.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View view) {
-                index = String.valueOf(view.getTag());
                 String stock_name = searchData.getStock_name();
+                String index = String.valueOf(searchData.getStock_index());
+                String id_san = searchData.getId_san();
                 Intent intent = new Intent(context, TradeActivity.class);
                 intent.putExtra("stockname", stock_name);
                 intent.putExtra("index", index);
-                intent.putExtra("id_san", "HOSE");
+                intent.putExtra("id_san", id_san);
```

Comment:
Please add a line of code like: `intent.setFlags(Intent.FLAG_ACTIVITY_NEW_TASK);` for calling startActivity outside of an Activity. It throws the exception that crashes the app